documentation/project-docs/external-component-telemetry.md

```diff
@@ -0,0 +1,39 @@
+# Responsibly managing telemetry in external components
+
+Many components are _delivered_ with or by the .NET SDK but want to collect and
+manage telemetry. The SDK has telemetry collection mechanisms that may appear
+attractive, but present down-sides for these authors.
+
+This document clarifies some guidelines for authors of components that are consumed by the .NET SDK but want to own their own telemetry.
```

Comment:
Should this be "specifies guidelines" or "provides guidelines". Clarifies some implies to me there are other guidelines we are not going to clarify.